Frontend/ui-driver/src/Services/APITypes.purs

```diff
@@ -1202,3 +1203,218 @@ instance showGetGetPerformanceRes :: Show GetPerformanceRes where show = generic
 instance standardEncodeGetPerformanceRes :: StandardEncode GetPerformanceRes where standardEncode (GetPerformanceRes req) = standardEncode req
 instance decodeGetPerformanceRes :: Decode GetPerformanceRes where decode = defaultDecode
 instance encodeGetPerformanceRes :: Encode GetPerformanceRes where encode = defaultEncode
+
+--------------------------------------------------- getCategories ----------------------------------------------------
+data GetCategoriesReq = GetCategoriesReq String
+
+newtype GetCategoriesRes = GetCategoriesRes
+  { categories :: Array
+    { label    :: String
+    , logoUrl  :: String
+    , category :: String
+    , issueCategoryId :: String
+    }
+  }
+
+instance makeGetCategoriesReq :: RestEndpoint GetCategoriesReq GetCategoriesRes where
+    makeRequest reqBody@(GetCategoriesReq language) headers = defaultMakeRequest GET (EP.getCategories language) headers reqBody
+    decodeResponse    = decodeJSON
+    encodeRequest req = defaultEncode req
+
+derive instance genericGetCategoriesReq :: Generic GetCategoriesReq _
+instance showGetCategoriesReq     :: Show GetCategoriesReq where show     = genericShow
+instance standardGetCategoriesReq :: StandardEncode GetCategoriesReq where standardEncode (GetCategoriesReq _) = standardEncode {}
+instance decodeGetCategoriesReq   :: Decode GetCategoriesReq where decode = defaultDecode
+instance encodeGetCategoriesReq   :: Encode GetCategoriesReq where encode = defaultEncode
+
+derive instance genericGetCategoriesRes :: Generic GetCategoriesRes _
+instance showGetGetCategoriesRes        :: Show GetCategoriesRes where show     = genericShow
+instance standardEncodeGetCategoriesRes :: StandardEncode GetCategoriesRes where standardEncode (GetCategoriesRes res) = standardEncode res
+instance decodeGetCategoriesRes         :: Decode GetCategoriesRes where decode = defaultDecode
+instance encodeGetCategoriesRes         :: Encode GetCategoriesRes where encode = defaultEncode
+
+--------------------------------------------------- getOptions ----------------------------------------------------
+data GetOptionsReq = GetOptionsReq String String
+
+newtype GetOptionsRes = GetOptionsRes
+  { options :: Array
```

Comment:
@ThisIsMani Can you create another newtype and then use instead of adding object directly

Reply:
Done